### DISCUSSION TITLE
Guthaben auf Firefly wird nicht mehr angezeigt

### DISCUSSION THREAD

**Discussion by tackar66** [iotaledger/firefly#8193] [Help & Support]
Hallo,
ich habe ein Problem ,
nach der Aktualisierung von Firefly.

Guthaben auf Firefly wird nicht mehr angezeigt, auch nach mehrmaligen Guthaben suche nicht, dafür aber 2 Sammlerstücke "you won 2000 Iota" wie bekomme ich meine ca.4000 Iota wieder?
Ich hoffe doch das man mir hier helfen kann.
Gruß

**Comment by begonaalvarezd** [comment]
Hi, did you enter your key in that iota lottery website? If so, its a scam and im afraid you have been robbed
